modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectStateFlowLocalServiceImpl.java

```diff
@@ -183,6 +183,48 @@ public ObjectStateFlow updateDefaultObjectStateFlow(
 		return null;
 	}
 
+	public void updateObjectStateFlowTransitions(
```

Comment:
@G-Sa can we move this method to the `ObjectStateTransitionLocalServiceImpl` to be in the same place as the already existing method `deleteObjectStateFlowObjectStateTransitions`?

Reply:
I don't know if it makes much sense. 
Moving the method there would mean one of two things, either splitting it in two, so we have the StateFlow part in this class and move the StateTransitions to the other, OR moving all logic to the `ObjectStateTransitionLocalServiceImpl` which would imply that the child objects (ObjectStateTransitions) would have power to change their "owner" (ObjectStateFlow) and that's usually a bad design idea.

Also, i made this method in this class because it already has other methods that deal with both flow and transitions (see `addObjectStateFlow`